konfera/models/event.py

```diff
@@ -53,6 +70,17 @@ def __str__(self):
     def duration(self):
         return (self.date_to - self.date_from).days + 1
 
+    @property
+    def cfp_open(self):
+        return self.cfp_allowed and self.cfp_end >= datetime.now(tz=timezone.utc)
+
+    def clean(self):
+        if self.cfp_allowed and self.cfp_end == datetime(2000, 1, 1, 0, 0, tzinfo=timezone.utc):
```

Comment:
this was the original intention, but during the tests it required default value, I did not realize I can easily use null=True to make it work